scripts/mock-api.mjs

```diff
@@ -7,12 +7,14 @@
  * - E2E tests will likely fail for value based on these mocks, so you'll need to update them.
  * - visual tests will fail too
  *
- * Last ran on 18th Sep 2023
+ * Last ran on 29th Feb 2024
```

Comment:
I would the the date for each function. Since you didn't updated the market price